docs/step-by-step/step-02.md

```diff
@@ -128,11 +117,10 @@ We built Netdata to be a big sandbox for learning more about your systems and ap
 Netdata's charts are fully interactive. You can pan through historical metrics, zoom in and out, select specific
 timeframes for further analysis, resize charts, and more.
 
-Best of all, Whenever you use a chart in this way, Netdata synchronizes all the other charts to match it. This even
-applies across different Netdata agents if you connect them using the [**My nodes** menu](/registry/README.md)!
+Best of all, Whenever you use a chart in this way, Netdata synchronizes all the other charts to match it.
 
-![Aniamted GIF of chart
-synchronziation](https://user-images.githubusercontent.com/1153921/63464271-c03a9c00-c413-11e9-971d-245238926193.gif)
+![Animated GIF of the standard Netdata dashboard being manipulated and synchronizing
```

Comment:
The gif doesn't seem to match the text - it shows resizing a chart rather than panning or zooming. There is another section (and gif) about resizing later - is this one correct?

Reply:
Fixed this. I mixed up all my gifs.